cmd/vic-machine/inspect/inspect.go

```diff
@@ -89,6 +97,17 @@ func (i *Inspect) Flags() []cli.Flag {
 	return flags
 }
 
+func (i *Inspect) ConfigFlags() []cli.Flag {
+	output := cli.StringFlag{
+		Name:        "output",
```

Comment:
As discussion with @ali5ter , output might be confusing if we need to write manifest file. Please change output to format